source/particle/property/interface.cc

```diff
@@ -731,6 +720,8 @@ namespace aspect
 
         // lastly store internal integrator properties
         property_list.emplace_back (std::make_unique<IntegratorProperties<dim>>());
+
+        property_list.back()->declare_parameters (prm);
```

Comment:
Why did this work before you added this line? Are the declare_parameters functions of the plugins called from somewhere else?